client/dist/style.css

```diff
@@ -1,10 +1,106 @@
+.title {
+  font-size: 40px;
+  display: flex;
+  flex-wrap: wrap;
+  font-family: Circular,-apple-system,BlinkMacSystemFont,Roboto,Helvetica Neue,sans-serif;
+}
+
+p {
+  margin-bottom: 5px;
+  font-family: Circular,-apple-system,BlinkMacSystemFont,Roboto,Helvetica Neue,sans-serif;
```

Comment:
@SimonC989 @carl12 This font is added to several different styles. Any time you find yourself writing the same thing in several places, there's probably some way to write it more efficiently. If we decide to change the font we're using, we'd have to change every one of those lines. Perhaps there's a way to apply the font to the whole component? It's worth figuring out how to write very dry CSS. CSS gets out of control very fast...